tests/applications/busybox/include/linux/netfilter/xt_connmark.h

```diff
@@ -1,42 +1,7 @@
-/* SPDX-License-Identifier: GPL-2.0+ WITH Linux-syscall-note */
```

Comment:
also wondering how this got changed and added?

Reply:
I don't know. It seems that these files were changed automatically on my laptop and I did a stage-all by accident. My bad. Should I just change them back?

Reply:
You should be able to just remove them from the commit. Look up how to do that with git.